### PR TITLE
Avoid post process if nothing has changed

### PR DESCRIPTION
**Pull request by sassanh**

### CODE REVIEW

File: pipeline/packager.py [hunk 1]
```diff
@@ -115,8 +117,11 @@ def pack(self, package, compress, signal, **kwargs):
             compiler_options=package.compiler_options,
             force=True,
         )
-        content = compress(paths, **kwargs)
-        self.save_file(output_filename, content)
+        output_path = self.storage.path(output_filename)
+        output_mtime = os.path.getmtime(output_path) if os.path.exists(output_path) else 0
+        if any([os.path.getmtime(self.storage.path(path)) >= output_mtime for path in paths]):
```
Comment (cyberdelia) [inline pipeline/packager.py]: I don’t think you can assume the storage files are the accessible using `os`, storage might expose  `get_modified_time` though.

Comment (sassanh) [reply pipeline/packager.py]: @cyberdelia thanks for review, In what cases it may not be accessible via `os`?

Comment (cyberdelia) [reply pipeline/packager.py]: If the storage points to a remote storage (i.e: something like s3, or else).

Comment (sassanh) [reply pipeline/packager.py]: In that case `output_mtime` will be `0`, and this code is ignored, but I modified it to use `storge.get_modified_time` to improve its compatibility.

**Pull request closed without merging**